docs/DEPLOYMENT.md

```diff
@@ -42,14 +49,22 @@ stringData:
   REFRESH_JWT_EXPIRES_IN: time such as 15m
 ```
 
-**NOTE:** DO NOT commit and push
+#### Config File
+
+If you want to change the config file passed to hub, you can edit the `30-api-configmap.yaml` and change the URL to the file. By default it is pointing to [config.yaml][config.yaml] in Hub which has Application Data.
+
+In this file we add users who have additional scopes which can be used to refresh a catalog, refresh config file after changes, create an agent token which can be used in cron job to refresh the catalog after an interval.
+
+All users have default scopes they get after login which allows them to rate the resources.
```

Comment:
Here you have mentioned that it allows to rate the resources, instead can we say or mention the default scopes are `rating:read` and `rating:write`